lib/Sabberworm/CSS/RuleSet/DeclarationBlock.php

```diff
@@ -26,7 +28,7 @@ public function __construct($iLineNo = 0) {
 		$this->aSelectors = array();
 	}
 
-	public static function parse(ParserState $oParserState) {
+	public static function parse(ParserState $oParserState, $oList = NULL) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method parse() has a Cyclomatic Complexity of 11. The configured cyclomatic complexity threshold is 10.](https://app.codacy.com/app/sabberworm/PHP-CSS-Parser/pullRequest?prid=4484969)